pom.xml

```diff
@@ -302,6 +303,27 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>flatten-maven-plugin</artifactId>
+				<version>1.6.0</version>
+				<configuration>
+					<outputDirectory>${project.build.directory}</outputDirectory>
```

Comment:
> .flattened-pom.xml is still missing in .gitignore.

We don't need that because the file is stored into a `target` directory which is already under ignore.
I don't see any argument why this file has to be in the root project dir when we are interested in it only at runtime on `install` or `deploy`, so that is indeed a volatile entity for project build lifecycle and that is totally enough to have it in the `target` alongside with the rest of artifacts to deliver.

Reply:
If you do that, you don't need `clean` then. Again, if you decide to do things differently, please do that consistently.

Reply:
Right. That’s what I did in the last commit. Please, double check the latest change set in the PR